velox/functions/sparksql/registration/RegisterDatetime.cpp

```diff
@@ -83,6 +84,25 @@ void registerDatetimeFunctions(const std::string& prefix) {
       {prefix + "make_ym_interval"});
   registerFunction<MakeYMIntervalFunction, IntervalYearMonth, int32_t, int32_t>(
       {prefix + "make_ym_interval"});
+  registerFunction<MakeDTIntervalFunction, IntervalDayTime>(
+      {prefix + "make_dt_interval"});
+  registerFunction<MakeDTIntervalFunction, IntervalDayTime, int32_t>(
+      {prefix + "make_dt_interval"});
+  registerFunction<MakeDTIntervalFunction, IntervalDayTime, int32_t, int32_t>(
+      {prefix + "make_dt_interval"});
+  registerFunction<
+      MakeDTIntervalFunction,
+      IntervalDayTime,
+      int32_t,
+      int32_t,
+      int32_t>({prefix + "make_dt_interval"});
```

Comment:
Spark would provide default values for the optional variables, so we might only need to register the four-arg one.